cvat/apps/webhooks/signals.py

```diff
@@ -0,0 +1,204 @@
+import hashlib
+import hmac
+import json
+
+import django_rq
+import requests
+from django.dispatch import Signal, receiver
+
+from cvat.apps.engine.models import Project
+from cvat.apps.engine.serializers import BasicUserSerializer
+from cvat.apps.organizations.models import Organization
+
+from .event_type import EventTypeChoice, event_name
+from .models import Webhook, WebhookDelivery, WebhookTypeChoice
+
+WEBHOOK_TIMEOUT = 10
+
+signal_update = Signal()
+signal_create = Signal()
+signal_delete = Signal()
+signal_redelivery = Signal()
+signal_ping = Signal()
+
+
+def send_webhook(webhook, payload, delivery):
+    headers = {}
+    if webhook.secret:
+        headers["X-Signature-256"] = "sha256=" + \
+            hmac.new(
+                webhook.secret.encode("utf-8"),
+                (json.dumps(payload) + "\n").encode("utf-8"),
+                digestmod=hashlib.sha256
+            ).hexdigest()
+
+    response = None
+    try:
+        response = requests.post(
+            webhook.target_url,
+            json=payload,
+            verify=webhook.enable_ssl,
+            headers=headers,
+            timeout=WEBHOOK_TIMEOUT
+        )
+        status_code = str(response.status_code)
+    except requests.ConnectionError:
+        status_code = "Failed to connect to target url"
```

Comment:
Let's have status_code and status_msg.

Reply:
I left only `status_code`. Now it will be always int value that indicate http status code.

In case when we have Connection error status_code is 502, and in case when we have timeout status_code is 504 similar to GitHub Webhooks

Having status message it's excessively, I think.